java-client/flight-examples/src/main/java/io/deephaven/client/examples/AddToInputTable.java

```diff
@@ -44,9 +54,10 @@ protected void execute(FlightSession flight) throws Exception {
 
             while (true) {
                 // Add a new row, at least once every second
-                final NewTable newRow = header.row(Instant.now()).newTable();
+                final NewTable newRow = header.row(true, (byte) 42, 'a', (short) 32_000, 1234567, 1234567890123L, 3.14f, 3.14d, "Hello, World",
+                      Instant.now(), "abc".getBytes()).newTable();
                 flight.addToInputTable(timestampHandle, newRow, bufferAllocator).get(5, TimeUnit.SECONDS);
-                Thread.sleep(ThreadLocalRandom.current().nextLong(1000));
+                Thread.sleep(ThreadLocalRandom.current().nextLong(100000));
```

Comment:
Let's set this value back to 1s. Alternatively, you could make this a flag.

Here is an example flag that is a similar type:
```java
    @Option(names = {"--input-table-size"}, description = "The input table size, default ${DEFAULT-VALUE}",
            defaultValue = "1000")
    int inputTableSize;
```
(Source for this flag is here: https://github.com/deephaven/deephaven-core/blob/main/java-client/flight-examples/src/main/java/io/deephaven/client/examples/AggInputBase.java#L30)

If you make as a flag; change the comment on line 56 to reference the flag rather than 'once every second'.

Reply:
reverted to 1s